pkg/trustedresources/verify.go

```diff
@@ -33,18 +33,33 @@ import (
 	"github.com/tektoncd/pipeline/pkg/trustedresources/verifier"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
+	"knative.dev/pkg/apis"
 	"knative.dev/pkg/logging"
 )
 
 const (
 	// SignatureAnnotation is the key of signature in annotation map
 	SignatureAnnotation = "tekton.dev/signature"
+	// ConditionTrustedResourcesVerified specifies that the resources pass trusted resources verification or not.
+	ConditionTrustedResourcesVerified apis.ConditionType = "TrustedResourcesVerified"
```

Comment:
should `ConditionTrustedResourcesVerified` be defined in this package?

Reply:
That's a good question! Maybe we could create a new pkg under trusted resources?

Reply:
on second thought I think this is probably fine.